src/sections/featured-collection.liquid

```diff
@@ -1,60 +1,66 @@
+<section class="featured-collection">
+  {% if section.settings.title != blank %}
+    <h2 class="featured-collection__title">{{ section.settings.title | escape }}</h2>
+  {% endif %}
 
-    {% if product.featured_image != blank %}
-      {% include 'responsive-image' with
-        image: product.featured_image,
-        max_width: 480,
-        max_height: 480
-      %}
-    {% endif %}
+  {%- assign collection = collections[section.settings.collection] -%}
 
-    <p>{{ product.title }}</p>
-  </a>
-
-  <p>
-    {% if product.compare_at_price > product.price %}
-
-      {% if product.price_varies %}
-        {%- assign sale_price = product.price | money -%}
-        {{ 'products.product.on_sale_from_html' | t: price: sale_price }}
-      {% else %}
-        {{ 'products.product.on_sale' | t }}
-        {{ product.price | money }}
-      {% endif %}
-
-    {% else %}
-
-      {% if product.price_varies %}
-        {%- assign price = product.price | money -%}
-        {{ 'products.product.from_text_html' | t: price: price }}
-      {% else %}
-        {{ product.price | money }}
+  {% for product in collection.products limit: 6 %}
+    <a href="{{ product.url | within: collection }}" class="featured-collection-item">
+      {% if product.featured_image != blank %}
+        <div class="featured-collection__image">
+          {% include 'responsive-image' with
+            image: product.featured_image,
+            max_width: 480,
+            max_height: 480
+          %}
+        </div>
       {% endif %}
 
-    {% endif %}
-  </p>
-{% else %}
-
-  {% for i in (1..6) %}
-    <a href="#">
-      {% capture current %}{% cycle 1, 2, 3, 4, 5, 6 %}{% endcapture %}
-      {{ 'product-' | append: current | placeholder_svg_tag: 'placeholder-svg placeholder-svg--small' }}
-
-      <p>
-        {{ 'homepage.onboarding.product_title' | t }}
-      </p>
+      <p>{{ product.title }}</p>
+      <span>
+        {% if product.compare_at_price > product.price %}
+          {% if product.price_varies %}
+            {%- assign sale_price = product.price | money -%}
+            {{ 'products.product.on_sale_from_html' | t: price: sale_price }}
+          {% else %}
+            {{ 'products.product.on_sale' | t }}
+            {{ product.price | money }}
+          {% endif %}
+        {% else %}
+          {% if product.price_varies %}
+            {%- assign price = product.price | money -%}
+            {{ 'products.product.from_text_html' | t: price: price }}
+          {% else %}
+            {{ product.price | money }}
+          {% endif %}
+        {% endif %}
+      </span>
     </a>
+  {% else %}
+    {% for i in (1..6) %}
+      <a href="#" class="featured-collection-item">
+        {% capture current %}{% cycle 1, 2, 3, 4, 5, 6 %}{% endcapture %}
+        {{ 'product-' | append: current | placeholder_svg_tag: 'placeholder-svg placeholder-svg--small' }}
 
-    <p>{{ 1999 | money }}</p>
+        <div class="featured-collection__info">
```

Comment:
We can remove this class and just have `<div>`.

Reply:
Thanks! Actually I removed the `<div>` because it's not needed.